Update cmake minimum version to 3.16

ndellingwood opened the pull request.

ndellingwood commented:
There is policy usage guarded by versions of cmake pre-3.17.1, but this is a start to increase minimum cmake version to align with Kokkos and Trilinos
kokkos/kokkos#3679
trilinos/Trilinos#8401

jjwilke commented on example/cmake/install/CMakeLists.txt:
```diff
@@ -1,6 +1,5 @@
+#Kokkos requires at least 3.17.1
+cmake_minimum_required (VERSION 3.17.1)
 
 IF(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.12.0")
```
This guard can be removed. Always true.

jjwilke commented:
@ndellingwood I pushed some more cleanup. I haven't had a chance to run spot checks on it, yet. That should be all the loose ends for 3.17, AFAICT.

jjwilke reviewed (changes requested):
Needs spot check with most recent changes. Squash to single commit after spot checks pass.

ndellingwood commented:
Spot-check in progress

jjwilke commented:
Still some debate swirling around 3.16 vs 3.17, I guess.  Won't change much, though.

ndellingwood commented:
@jjwilke current version of PR passed spot-check on kokkos-dev-2 (with required cmake version update in testing scripts that I merged from develop), thanks for updating the PR


```bash
Running on machine: kokkos-dev-2

KokkosKernels Repository Status:  72263fa9992f36430003fb8545b57a340662b834 Merge branch 'develop' into cmake-version-update

Kokkos Repository Status:  8c241078d0b844633cb1eae186bbb6a8a810ff5d Merge pull request #3681 from ndellingwood/update-testall

#######################################################
PASSED TESTS
#######################################################
clang-8.0-Cuda_OpenMP-release build_time=731 run_time=113
clang-8.0-Pthread_Serial-release build_time=244 run_time=134
clang-9.0.0-Pthread-release build_time=147 run_time=62
clang-9.0.0-Serial-release build_time=165 run_time=50
cuda-10.1-Cuda_OpenMP-release build_time=915 run_time=114
cuda-11.0-Cuda_OpenMP-release build_time=929 run_time=113
cuda-9.2-Cuda_Serial-release build_time=883 run_time=200
gcc-7.3.0-OpenMP-release build_time=169 run_time=47
gcc-7.3.0-Pthread-release build_time=135 run_time=62
gcc-8.3.0-Serial-release build_time=166 run_time=52
gcc-9.1-OpenMP-release build_time=213 run_time=48
gcc-9.1-Serial-release build_time=194 run_time=52
intel-17.0.1-Serial-release build_time=390 run_time=55
intel-18.0.5-OpenMP-release build_time=690 run_time=50
intel-19.0.5-Pthread-release build_time=347 run_time=64

```

ndellingwood commented:
Updated title and PR for CMake min version to 3.16

jjwilke reviewed (dismissed):
This looks good. The only hiccup would be cmake/kokkos_tribits.cmake:408 and following... We can now correctly deduplicate flags, which was not possible before. If I have time tonight, I can try to push that.  This uses the cmake SHELL syntax.

We should also squash to 1 commit.

e10harvey reviewed (changes requested):
One more cmake_minimum_version update in `install_test/CMakeLists.txt.in`.

ndellingwood commented:
Thanks for feedback, I made another update and rebased to a single commit.

@jjwilke what changes are needed here to accommodate for the kokkos flag deduplication stuff you mentioned in kokkos at `cmake/kokkos_tribits.cmake:408`?

@e10harvey thanks for the catch, I rebased on top of develop and made the requested change

lucbv commented:
Good to hear the rebase went well, I think it would be clean if we can have this in the release.

lucbv commented:
@e10harvey if you have a minute, can you check this PR?
Also do you know how to get the auto-tester to pick it up, it has the RETEST label for a few days but never got tested?

e10harvey commented:
> @e10harvey if you have a minute, can you check this PR?
> Also do you know how to get the auto-tester to pick it up, it has the RETEST label for a few days but never got tested?

Yea, the auto-tester should pick it up later today.

e10harvey reviewed (approved):
Thank you, Nathan.

kokkos-devops-admin commented:
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection Is Not Necessary for this Pull Request.

kokkos-devops-admin commented:
Status Flag 'Pull Request AutoTester' - User Requested Retest - Label AT: RETEST will be reset after testing.

kokkos-devops-admin commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA9_Tpls_CUDA10_Tpls_CUDA10_LayoutRight_GCC720_Light_GCC720_GCC740</h3>
<ul>
  <li>Build Num: 68</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>cmake-version-update</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>5fdb9d7577d0e9f8b153f137c0a060cffd1b72ec</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>a674aad01b6aa56e36651554ab4198c63952f2a8</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;BlocksPromotion</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>866</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720</h3>
<ul>
  <li>Build Num: 839</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>cmake-version-update</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>5fdb9d7577d0e9f8b153f137c0a060cffd1b72ec</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>a674aad01b6aa56e36651554ab4198c63952f2a8</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;BlocksPromotion</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>866</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720_Light_LayoutRight</h3>
<ul>
  <li>Build Num: 486</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>cmake-version-update</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>5fdb9d7577d0e9f8b153f137c0a060cffd1b72ec</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>a674aad01b6aa56e36651554ab4198c63952f2a8</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;BlocksPromotion</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>866</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC720</h3>
<ul>
  <li>Build Num: 830</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>cmake-version-update</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>5fdb9d7577d0e9f8b153f137c0a060cffd1b72ec</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>a674aad01b6aa56e36651554ab4198c63952f2a8</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;BlocksPromotion</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>866</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_INTEL18</h3>
<ul>
  <li>Build Num: 818</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>cmake-version-update</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>5fdb9d7577d0e9f8b153f137c0a060cffd1b72ec</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>a674aad01b6aa56e36651554ab4198c63952f2a8</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;BlocksPromotion</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>866</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CLANG1001</h3>
<ul>
  <li>Build Num: 223</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>cmake-version-update</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>5fdb9d7577d0e9f8b153f137c0a060cffd1b72ec</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>a674aad01b6aa56e36651554ab4198c63952f2a8</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;BlocksPromotion</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>866</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: KOKKOSKERNELS (ndellingwood/kokkos-kernels)</dt>
  <ul>
    <li>Branch: cmake-version-update</li>
    <li>SHA: 5fdb9d7577d0e9f8b153f137c0a060cffd1b72ec</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: ndellingwood
</details>

kokkos-devops-admin commented:
Status Flag 'Pull Request AutoTester' - Jenkins Testing: all Jobs PASSED

<details>
<summary>
  <b>Pull Request Auto Testing has PASSED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA9_Tpls_CUDA10_Tpls_CUDA10_LayoutRight_GCC720_Light_GCC720_GCC740</h3>
<ul>
  <li>Build Num: 68</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>cmake-version-update</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>5fdb9d7577d0e9f8b153f137c0a060cffd1b72ec</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>a674aad01b6aa56e36651554ab4198c63952f2a8</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;BlocksPromotion</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>866</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720</h3>
<ul>
  <li>Build Num: 839</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>cmake-version-update</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>5fdb9d7577d0e9f8b153f137c0a060cffd1b72ec</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>a674aad01b6aa56e36651554ab4198c63952f2a8</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;BlocksPromotion</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>866</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720_Light_LayoutRight</h3>
<ul>
  <li>Build Num: 486</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>cmake-version-update</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>5fdb9d7577d0e9f8b153f137c0a060cffd1b72ec</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>a674aad01b6aa56e36651554ab4198c63952f2a8</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;BlocksPromotion</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>866</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC720</h3>
<ul>
  <li>Build Num: 830</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>cmake-version-update</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>5fdb9d7577d0e9f8b153f137c0a060cffd1b72ec</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>a674aad01b6aa56e36651554ab4198c63952f2a8</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;BlocksPromotion</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>866</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_INTEL18</h3>
<ul>
  <li>Build Num: 818</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>cmake-version-update</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>5fdb9d7577d0e9f8b153f137c0a060cffd1b72ec</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>a674aad01b6aa56e36651554ab4198c63952f2a8</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;BlocksPromotion</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>866</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CLANG1001</h3>
<ul>
  <li>Build Num: 223</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>cmake-version-update</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/ndellingwood/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>5fdb9d7577d0e9f8b153f137c0a060cffd1b72ec</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>a674aad01b6aa56e36651554ab4198c63952f2a8</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;BlocksPromotion</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>866</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>
</details>

kokkos-devops-admin commented:
Status Flag 'Pre-Merge Inspection' - SUCCESS: The last commit to this Pull Request has been INSPECTED AND APPROVED by [ e10harvey lucbv ]!

kokkos-devops-admin commented:
Status Flag 'Pull Request AutoTester' - Pull Request MUST BE MERGED MANUALLY BY Project Team - This Repo does not support Automerge
